src/toil_scripts/tools/variant_filters.py

```diff
@@ -0,0 +1,270 @@
+
+    outputs = {'joint.vcf': None}
+    docker_call(work_dir = work_dir,
+                env={'JAVA_OPTS':'-Djava.io.tmpdir=/data/ -Xmx{}'.format(config.xmx)},
+                parameters = command,
+                tool = 'quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
+                inputs=inputs.keys(),
+                outputs=outputs)
+    return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'joint.vcf'))
+
+
+def gatk_select_variants(job, mode, vcf_id, config):
+    """
+    Isolate variant types using GATK SelectVariants
+
+    :param job: Toil Job instance
+    :param mode str: variant type (i.e. SNP or INDEL)
+    :param vcf_id str: VCF FileStoreID
+    :param config Namespace: Configuration and shared FileStoreIDs
+    :return: VCF FileStoreID
+    :rtype: str
+    """
+    job.fileStore.logToMaster('Running GATK SelectVariants: %s' % mode)
+    work_dir = job.fileStore.getLocalTempDir()
+    inputs = {'genome.fa': config.genome_fasta, 'genome.fa.fai': config.genome_fai,
+              'genome.dict': config.genome_dict, 'input.vcf': vcf_id}
+    get_files_from_filestore(job, work_dir, inputs)
+
+    output = "raw_%s.vcf" % mode
+
+    command = ['-R', 'genome.fa',
+               '-T', 'SelectVariants',
+               '-V', 'input.vcf',
+               '-o', output,
+               '-selectType', mode]
+
+    outputs = {output: None}
+    docker_call(work_dir = work_dir,
+                env={'_JAVA_OPTIONS':'-Djava.io.tmpdir=/data/ -Xmx{}'.format(config.xmx)},
+                parameters = command,
+                tool = 'quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
+                inputs=inputs.keys(),
+                outputs=outputs)
+    return job.fileStore.writeGlobalFile(os.path.join(work_dir, output))
+
+
+def gatk_variant_filtration(job, mode, vcf_id, config):
+    """
+    Filters VCF using GATK recommended filters. VCF must contain a single variant
+    type: SNPs or INDEL.
+
+    SNP Filter:
+    QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0
+
+    INDEL Filter:
+    QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0
+
+    :param job: Toil Job instance
+    :param mode str: variant type
+    :param vcf_id str: VCF FileStoreID
+    :param config Namespace: Configuration and shared FileStoreIDs
+    :return: Filtered VCF FileStoreID
+    :rtype: str
+    """
+    job.fileStore.logToMaster('Apply %s Filter' % mode)
+    work_dir = job.fileStore.getLocalTempDir()
+    inputs = {'genome.fa': config.genome_fasta, 'genome.fa.fai': config.genome_fai,
+              'genome.dict': config.genome_dict, 'input.vcf': vcf_id}
+    get_files_from_filestore(job, work_dir, inputs)
+
+    # Recommended GATK hard filters:
+    # https://software.broadinstitute.org/gatk/documentation/article?id=2806
+    mode = mode.upper()
+    if mode == 'SNP':
+        expression = '"QD < 2.0 || FS > 60.0 || MQ < 40.0 || ' \
```

Comment:
I feel like these filter expressions should be pushed into the config file so that they're easy to modify. This might be a good thing to triage for now and open a ticket for.